### ISSUE TITLE
2025-04-16, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#142]
# Today's Arxiv Papers

(1) [DeepMath-103K: A Large-Scale, Challenging, Decontaminated, and Verifiable Mathematical Dataset for Advancing Reasoning](http://arxiv.org/abs/2504.11456v1), 2025-04-15

(2) [TextArena](http://arxiv.org/abs/2504.11442v1), 2025-04-15

(3) [Masculine Defaults via Gendered Discourse in Podcasts and Large Language Models](http://arxiv.org/abs/2504.11431v1), 2025-04-15

(4) [A Dual-Space Framework for General Knowledge Distillation of Large Language Models](http://arxiv.org/abs/2504.11426v1), 2025-04-15

(5) [Reinforcing Compositional Retrieval: Retrieving Step-by-Step for Composing Informative Contexts](http://arxiv.org/abs/2504.11420v1), 2025-04-15

(6) [Efficient Hybrid Language Model Compression through Group-Aware SSM Pruning](http://arxiv.org/abs/2504.11409v1), 2025-04-15

(7) [DataDecide: How to Predict Best Pretraining Data with Small Experiments](http://arxiv.org/abs/2504.11393v1), 2025-04-15

(8) [RankAlign: A Ranking View of the Generator-Validator Gap in Large Language Models](http://arxiv.org/abs/2504.11381v1), 2025-04-15

(9) [Cancer-Myth: Evaluating AI Chatbot on Patient Questions with False Presuppositions](http://arxiv.org/abs/2504.11373v1), 2025-04-15

(10) [OpenTuringBench: An Open-Model-based Benchmark and Framework for Machine-Generated Text Detection and Attribution](http://arxiv.org/abs/2504.11369v1), 2025-04-15

(11) [Teaching Large Language Models to Reason through Learning and Forgetting](http://arxiv.org/abs/2504.11364v1), 2025-04-15

(12) [DataSentinel: A Game-Theoretic Detection of Prompt Injection Attacks](http://arxiv.org/abs/2504.11358v1), 2025-04-15

(13) [Kimina-Prover Preview: Towards Large Formal Reasoning Models with Reinforcement Learning](http://arxiv.org/abs/2504.11354v1), 2025-04-15

(14) [A Minimalist Approach to LLM Reasoning: from Rejection Sampling to Reinforce](http://arxiv.org/abs/2504.11343v1), 2025-04-15

(15) [Optimizing LLM Inference: Fluid-Guided Online Scheduling with Memory Constraints](http://arxiv.org/abs/2504.11320v1), 2025-04-15

(16) [Learning to Be A Doctor: Searching for Effective Medical Agent Architectures](http://arxiv.org/abs/2504.11301v1), 2025-04-15

(17) [Automated Python Translation](http://arxiv.org/abs/2504.11290v1), 2025-04-15

(18) [The Obvious Invisible Threat: LLM-Powered GUI Agents' Vulnerability to Fine-Print Injections](http://arxiv.org/abs/2504.11281v1), 2025-04-15

(19) [From Misleading Queries to Accurate Answers: A Three-Stage Fine-Tuning Method for LLMs](http://arxiv.org/abs/2504.11277v1), 2025-04-15

(20) [Distillation-Supervised Convolutional Low-Rank Adaptation for Efficient Image Super-Resolution](http://arxiv.org/abs/2504.11271v1), 2025-04-15

(21) [Towards Automated Safety Requirements Derivation Using Agent-based RAG](http://arxiv.org/abs/2504.11243v1), 2025-04-15

(22) [Nondeterministic Polynomial-time Problem Challenge: An Ever-Scaling Reasoning Benchmark for LLMs](http://arxiv.org/abs/2504.11239v1), 2025-04-15

(23) [AutoRAN: Automated and Zero-Touch Open RAN Systems](http://arxiv.org/abs/2504.11233v1), 2025-04-15

(24) [Video Summarization with Large Language Models](http://arxiv.org/abs/2504.11199v1), 2025-04-15

(25) [Enhancing multimodal analogical reasoning with Logic Augmented Generation](http://arxiv.org/abs/2504.11190v1), 2025-04-15

(26) [Benchmarking Next-Generation Reasoning-Focused Large Language Models in Ophthalmology: A Head-to-Head Evaluation on 5,888 Items](http://arxiv.org/abs/2504.11186v1), 2025-04-15

(27) [Exploring Backdoor Attack and Defense for LLM-empowered Recommendations](http://arxiv.org/abs/2504.11182v1), 2025-04-15

(28) [MuSeD: A Multimodal Spanish Dataset for Sexism Detection in Social Media Videos](http://arxiv.org/abs/2504.11169v1), 2025-04-15

(29) [Bypassing Prompt Injection and Jailbreak Detection in LLM Guardrails](http://arxiv.org/abs/2504.11168v1), 2025-04-15

(30) [Fine-Tuning Large Language Models on Quantum Optimization Problems for Circuit Generation](http://arxiv.org/abs/2504.11109v1), 2025-04-15

(31) [Benchmarking Vision Language Models on German Factual Data](http://arxiv.org/abs/2504.11108v1), 2025-04-15

(32) [Using LLMs as prompt modifier to avoid biases in AI image generators](http://arxiv.org/abs/2504.11104v1), 2025-04-15

(33) [Consensus Entropy: Harnessing Multi-VLM Agreement for Self-Verifying and Self-Improving OCR](http://arxiv.org/abs/2504.11101v1), 2025-04-15

(34) [Evaluation Report on MCP Servers](http://arxiv.org/abs/2504.11094v1), 2025-04-15

(35) [QAMA: Quantum annealing multi-head attention operator with classical deep learning framework](http://arxiv.org/abs/2504.11083v1), 2025-04-15

(36) [DeepMLF: Multimodal language model with learnable tokens for deep fusion in sentiment analysis](http://arxiv.org/abs/2504.11082v1), 2025-04-15

(37) [DPS: Design Pattern Summarisation Using Code Features](http://arxiv.org/abs/2504.11081v1), 2025-04-15

(38) [Leveraging LLMs and attention-mechanism for automatic annotation of historical maps](http://arxiv.org/abs/2504.11050v1), 2025-04-15

(39) [LazyReview A Dataset for Uncovering Lazy Thinking in NLP Peer Reviews](http://arxiv.org/abs/2504.11042v1), 2025-04-15

(40) [MMC: Iterative Refinement of VLM Reasoning via MCTS-based Multimodal Critique](http://arxiv.org/abs/2504.11009v1), 2025-04-15

(41) [Dynamic Compressing Prompts for Efficient Inference of Large Language Models](http://arxiv.org/abs/2504.11004v1), 2025-04-15

(42) [Dopamine Audiobook: A Training-free MLLM Agent for Emotional and Human-like Audiobook Generation](http://arxiv.org/abs/2504.11002v1), 2025-04-15

(43) [ReZero: Enhancing LLM search ability by trying one-more-time](http://arxiv.org/abs/2504.11001v1), 2025-04-15

(44) [Exploring the Role of KG-Based RAG in Japanese Medical Question Answering with Small-Scale LLMs](http://arxiv.org/abs/2504.10982v1), 2025-04-15

(45) [When is Task Vector Provably Effective for Model Editing? A Generalization Analysis of Nonlinear Transformers](http://arxiv.org/abs/2504.10957v1), 2025-04-15

(46) [Unveiling Challenges for LLMs in Enterprise Data Engineering](http://arxiv.org/abs/2504.10950v1), 2025-04-15

(47) [Can LLMs Leverage Observational Data? Towards Data-Driven Causal Discovery with LLMs](http://arxiv.org/abs/2504.10936v1), 2025-04-15

(48) [MSCRS: Multi-modal Semantic Graph Prompt Learning Framework for Conversational Recommender Systems](http://arxiv.org/abs/2504.10921v1), 2025-04-15

(49) [Adaptive Human-Agent Teaming: A Review of Empirical Studies from the Process Dynamics Perspective](http://arxiv.org/abs/2504.10918v1), 2025-04-15

(50) [Towards A Universal Graph Structural Encoder](http://arxiv.org/abs/2504.10917v1), 2025-04-15

(51) [Understanding LLMs' Cross-Lingual Context Retrieval: How Good It Is And Where It Comes From](http://arxiv.org/abs/2504.10906v1), 2025-04-15

(52) [Leveraging Submodule Linearity Enhances Task Arithmetic Performance in LLMs](http://arxiv.org/abs/2504.10902v1), 2025-04-15

(53) [Xpose: Bi-directional Engineering for Hidden Query Extraction](http://arxiv.org/abs/2504.10898v1), 2025-04-15

(54) [ARise: Towards Knowledge-Augmented Reasoning via Risk-Adaptive Search](http://arxiv.org/abs/2504.10893v1), 2025-04-15

(55) [Exploring Persona-dependent LLM Alignment for the Moral Machine Experiment](http://arxiv.org/abs/2504.10886v1), 2025-04-15

(56) [Large Language Model-Informed Feature Discovery Improves Prediction and Interpretation of Credibility Perceptions of Visual Content](http://arxiv.org/abs/2504.10878v1), 2025-04-15

(57) [Enhancing Features in Long-tailed Data Using Large Vision Mode](http://arxiv.org/abs/2504.10852v1), 2025-04-15

(58) [Moving Beyond Next-Token Prediction: Transformers are Context-Sensitive Language Generators](http://arxiv.org/abs/2504.10845v1), 2025-04-15

(59) [Rethinking Theory of Mind Benchmarks for LLMs: Towards A User-Centered Perspective](http://arxiv.org/abs/2504.10839v1), 2025-04-15

(60) [LayoutCoT: Unleashing the Deep Reasoning Potential of Large Language Models for Layout Generation](http://arxiv.org/abs/2504.10829v1), 2025-04-15

(61) [CLASH: Evaluating Language Models on Judging High-Stakes Dilemmas from Multiple Perspectives](http://arxiv.org/abs/2504.10823v1), 2025-04-15

(62) [CSPLADE: Learned Sparse Retrieval with Causal Language Models](http://arxiv.org/abs/2504.10816v1), 2025-04-15

(63) [Tabular foundation model to detect empathy from visual cues](http://arxiv.org/abs/2504.10808v1), 2025-04-15

(64) [Name of Thrones: Evaluating How LLMs Rank Student Names, Race, and Gender in Status Hierarchies](http://arxiv.org/abs/2504.10797v1), 2025-04-15

(65) [GUM-SAGE: A Novel Dataset and Approach for Graded Entity Salience Prediction](http://arxiv.org/abs/2504.10792v1), 2025-04-15

(66) [Can Large Language Models Trade? Testing Financial Theories with LLM Agents in Market Simulations](http://arxiv.org/abs/2504.10789v1), 2025-04-15

(67) [ATLASv2: LLM-Guided Adaptive Landmark Acquisition and Navigation on the Edge](http://arxiv.org/abs/2504.10784v1), 2025-04-15

(68) [The Art of Audience Engagement: LLM-Based Thin-Slicing of Scientific Talks](http://arxiv.org/abs/2504.10768v1), 2025-04-15

(69) [How Instruction and Reasoning Data shape Post-Training: Data Quality through the Lens of Layer-wise Gradients](http://arxiv.org/abs/2504.10766v1), 2025-04-14

(70) [CleanMAP: Distilling Multimodal LLMs for Confidence-Driven Crowdsourced HD Map Updates](http://arxiv.org/abs/2504.10738v1), 2025-04-14

(71) [Foundation Models for Remote Sensing: An Analysis of MLLMs for Object Localization](http://arxiv.org/abs/2504.10727v1), 2025-04-14

(72) [HELIOS: Adaptive Model And Early-Exit Selection for Efficient LLM Inference Serving](http://arxiv.org/abs/2504.10724v1), 2025-04-14

(73) [Can LLMs Classify CVEs? Investigating LLMs Capabilities in Computing CVSS Vectors](http://arxiv.org/abs/2504.10713v1), 2025-04-14

(74) [Legally-Informed Explainable AI](http://dx.doi.org/10.5281/zenodo.15170444), 2025-04-14

(75) [GestureCoach: Rehearsing for Engaging Talks with LLM-Driven Gesture Recommendations](http://arxiv.org/abs/2504.10706v1), 2025-04-14

(76) [The Jailbreak Tax: How Useful are Your Jailbreak Outputs?](http://arxiv.org/abs/2504.10694v1), 2025-04-14

(77) [Load Balancing with Network Latencies via Distributed Gradient Descent](http://arxiv.org/abs/2504.10693v1), 2025-04-14

(78) [Introducing Large Language Models as the Next Challenging Internet Traffic Source](http://arxiv.org/abs/2504.10688v1), 2025-04-14

(79) [EMAFusion: A Self-Optimizing System for Seamless LLM Selection and Integration](http://arxiv.org/abs/2504.10681v1), 2025-04-14

(80) [LITERA: An LLM Based Approach to Latin-to-English Translation](http://arxiv.org/abs/2504.10660v1), 2025-04-14

(81) [Relation-Rich Visual Document Generator for Visual Information Extraction](http://arxiv.org/abs/2504.10659v1), 2025-04-14

(82) [Un marco conceptual para la generación de requerimientos de software de calidad](http://arxiv.org/abs/2504.10654v1), 2025-04-14

(83) [Weight-of-Thought Reasoning: Exploring Neural Network Weights for Enhanced LLM Reasoning](http://arxiv.org/abs/2504.10646v1), 2025-04-14

(84) [Who is More Bayesian: Humans or ChatGPT?](http://arxiv.org/abs/2504.10636v1), 2025-04-14

(85) [Beyond Chains of Thought: Benchmarking Latent-Space Reasoning Abilities in Large Language Models](http://arxiv.org/abs/2504.10615v1), 2025-04-14

(86) [Demo: ViolentUTF as An Accessible Platform for Generative AI Red Teaming](http://arxiv.org/abs/2504.10603v1), 2025-04-14

(87) [xVerify: Efficient Answer Verifier for Reasoning Model Evaluations](http://arxiv.org/abs/2504.10481v1), 2025-04-14

(88) [InternVL3: Exploring Advanced Training and Test-Time Recipes for Open-Source Multimodal Models](http://arxiv.org/abs/2504.10479v2), 2025-04-14

(89) [MIEB: Massive Image Embedding Benchmark](http://arxiv.org/abs/2504.10471v1), 2025-04-14

(90) [Pixel-SAIL: Single Transformer For Pixel-Grounded Understanding](http://arxiv.org/abs/2504.10465v1), 2025-04-14

(91) [The Scalability of Simplicity: Empirical Analysis of Vision-Language Learning with a Single Transformer](http://arxiv.org/abs/2504.10462v1), 2025-04-14

(92) [GUI-R1 : A Generalist R1-Style Vision-Language Action Model For GUI Agents](http://arxiv.org/abs/2504.10458v2), 2025-04-14

(93) [M1: Towards Scalable Test-Time Compute with Mamba Reasoning Models](http://arxiv.org/abs/2504.10449v1), 2025-04-14

(94) [Multimodal Long Video Modeling Based on Temporal Dynamic Context](http://arxiv.org/abs/2504.10443v1), 2025-04-14

(95) [LLM Can be a Dangerous Persuader: Empirical Study of Persuasion Safety in Large Language Models](http://arxiv.org/abs/2504.10430v1), 2025-04-14

(96) [Can We Edit LLMs for Long-Tail Biomedical Knowledge?](http://arxiv.org/abs/2504.10421v1), 2025-04-14

(97) [Unchecked and Overlooked: Addressing the Checkbox Blind Spot in Large Language Models with CheckboxQA](http://arxiv.org/abs/2504.10419v2), 2025-04-14

(98) [CliniChat: A Multi-Source Knowledge-Driven Framework for Clinical Interview Dialogue Reconstruction and Evaluation](http://arxiv.org/abs/2504.10418v1), 2025-04-14

(99) [LLM-SRBench: A New Benchmark for Scientific Equation Discovery with Large Language Models](http://arxiv.org/abs/2504.10415v1), 2025-04-14

(100) [HUMOTO: A 4D Dataset of Mocap Human Object Interactions](http://arxiv.org/abs/2504.10414v1), 2025-04-14
